plugins/inputs/procstat/filter.go

```diff
@@ -180,7 +179,7 @@ func (f *Filter) ApplyFilter() ([]processGroup, error) {
 
 			matched = append(matched, p)
 		}
-		result = append(result, processGroup{processes: matched, tags: g.tags})
+		result = append(result, processGroup{processes: matched, tags: g.tags, level: 0})
```

Comment:
`level` will be `0` without explicitly setting it. Please revert to the previous code.